040-SDK/135-sql-access.mdx

```diff
@@ -193,6 +197,34 @@ xata.sql().query(
 
 </TabbedCode>
 
+### Query records from the replica store
+
+<TabbedCode tabs={['TypeScript', 'Python', 'JSON']}>
+```ts
+const response = await xata.sql<UsersRecord>({
+  statement: `SELECT * FROM "Users" LIMIT 10;`,
```

Comment:
```suggestion
  statement: `SELECT * FROM "\Users\" LIMIT 10;`,
```

I don't know if under the hood the TS SDK adds the `\`, if not they must be added to the table name, as otherwise the statement is not valid

Reply:
ping @SferaDev ^^

Reply:
We scape them yeah, no need to do anything here